### ISSUE TITLE
ValueError when generating a reaction network

### ISSUE THREAD

**Issue by HiroumiTani** [ReactionMechanismGenerator/RMG-Py#1300]
A job with Pdep crashes when generating a reaction network with the following message. I haven't detect from which commit this issue happens, but I am sure that rmg.py at the commit of e32da33dd628c9386ffeffdf2e439a316c60387c didn't have this problem. Please let me know if you have an idea on this issue. Thank you.

```
Generating kinetics for new reactions...
Updating 1 modified unimolecular reaction networks (out of 1)...
Updating PDepNetwork #1
Warning: Statmech node 'N_R0' and all its parents have data = None
Warning: Statmech node 'N_R0' and all its parents have data = None
Warning: Maximum number of iterations reached when fitting spectral data for NN.
For reaction hydrazine(1) <=> H(8) + N2H3(58), Ea raised from 320.8 to 338.9 kJ/mol to match endothermicity of reaction.
========================================================================
1 network information
---------------------
Isomers:
    hydrazine(1)                                          86.2423 kJ/mol
Reactant channels:
Product channels:
    NH2(46) + NH2(46)                                     352.223 kJ/mol
    H(8) + N2H3(58)                                       425.149 kJ/mol
Path reactions:
    hydrazine(1) <=> NH2(46) + NH2(46)                     364.75 kJ/mol
    hydrazine(1) <=> H(8) + N2H3(58)                      425.149 kJ/mol
========================================================================

Calculating densities of states for 1 network...
Using 513 grains from 86.24 to 1157.35 kJ/mol in steps of 2.09 kJ/mol to compute densities of states
Traceback (most recent call last):
  File "/home/hiroumi/rmg/RMG-Py/rmg.py", line 173, in <module>
    main()
  File "/home/hiroumi/rmg/RMG-Py/rmg.py", line 167, in main
    rmg.execute(**kwargs)
  File "/home/hiroumi/rmg/RMG-Py/rmgpy/rmg/main.py", line 567, in execute
    bimolecularReact=self.bimolecularReact)
  File "/home/hiroumi/rmg/RMG-Py/rmgpy/rmg/model.py", line 686, in enlarge
    self.updateUnimolecularReactionNetworks()
  File "/home/hiroumi/rmg/RMG-Py/rmgpy/rmg/model.py", line 1659, in updateUnimolecularReactionNetworks
    network.update(self, self.pressureDependence)
  File "/home/hiroumi/rmg/RMG-Py/rmgpy/rmg/pdep.py", line 557, in update
    self.initialize(Tmin, Tmax, Pmin, Pmax, maximumGrainSize, minimumGrainCount, activeJRotor, activeKRotor, rmgmode)
  File "/home/hiroumi/rmg/RMG-Py/rmgpy/pdep/network.py", line 222, in initialize
    logging.debug('The nework now has values of {0}'.format(repr(self)))
  File "/home/hiroumi/rmg/RMG-Py/rmgpy/pdep/network.py", line 140, in __repr__
    if self.E0: string += 'E0="{0}", '.format(self.E0)
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

**Comment by alongd** [comment]
I also got this on the master benchmark branch while running tests (eg3)

**Comment by alongd** [comment]
@goldmanm , could you check the `if self.E0` condition, performed on a list, in network.py?

**Comment by mjohnson541** [comment]
I took a look at this, self.E0 is indeed suppose to be an array.  I think the intent in the repr method was to check that it wasn't None, but it seems to be checking the truth value as a whole instead so when it calls the repr method it errors I think this is a side effect of #1145.

**Comment by mliu49** [comment]
Fixed by #1303.
